linchpin/__init__.py

```diff
@@ -122,6 +122,8 @@ def runcli(ctx, config, workspace, verbose, version, creds_path):
 
     if workspace is not None:
         ctx.workspace = os.path.realpath(os.path.expanduser(workspace))
+    else:
```

Comment:
I have checked the latest code and I can't find the setting of workplace in the initialisation of the LinchpinContext at https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/linchpin/api/context.py#L35.

Further investigation shows that this commit https://github.com/CentOS-PaaS-SIG/linchpin/pull/320/commits/d8e9ba136492dd44bd65881851038868a53b7a9a removes this set.

If the line:
self.workspace = os.path.realpath(os.path.curdir)

is added back to the __init__ this would fix the this current issue.